docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -94,24 +94,34 @@ all labeled and additional information will be included below.
 .. image:: pregnancy_decision_tree_vr.svg
 
 .. list-table:: Pregnancy Decision Tree
-  :widths: 3 15 15
+  :widths: 3 5 10 10 15
   :header-rows: 1
 
   * - ID
     - Decision Information 
+    - Data Value 
+    - Source
     - Notes
   * - 1
-    - XX% of simulants will receive ANC care, based on their propensity 
-    - Several details are still outstanding including: will ANC vary with age, subnational location or upstream factors, will ANC care propensity be correlated with delivery facility propensity
+    - ANC1 rates
+    - get_covariate_estimates(location_id=location_id, gbd_round_id=7, year_id=2021, decomp_step='iterative', covariate_id=7)
+    - GBD covariate ID 7
+    - This is location specific, but not age specific. Currently assume that there is no correlation of ANC with other factors.
   * - 2
-    - XX% of simulants will receive AI-assisted ultrasound, XX% will receive standard ultrasound, XX% will receive no ultrasound 
-    - Need to determine if this is random or correlated with other outcomes. Need to find baseline and scenario values.
+    - Ultrasound rate at ANC in baseline scenario
+    - 60.7% 
+    - https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9201203/
+    - Data value is a stand-in as we finalize. We would like this to be location specific, please code accordingly.
   * - 3
-    - All patients are assigned a stated gestational age (GA). Their stated GA will be their real GA +/- an error margin randomly assigned within an interval. For AI assisted ultrasound, the interval is +/- XX weeks, for standard ultrasound XX weeks, for no ultrasound XX weeks. 
-    - Check if distribution for error is random or more likely to be an over/underestimate. 
+    - Stated gestational age (GA) at ANC 
+    - Real GA +/- a value from a uniform distribution of: 0-5 days for AI ultrasound, 10-30 days for standard ultrasound, 10-45.5 days for no ultrasound 
+    - Ultrasound values assumed from BMGF inputs. No ultrasound value from this paper: https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0272718#sec007 
```

Comment:
This comment confused me: At first I thought it meant "There are no values for ultrasounds found in this paper," which seems strange because I'm more interested in what _is_ in the paper than what _isn't_ in it. But then I realized that it meant that "the values wer're using for the 'no ultrasound' case were taken from this paper," I think.

Reply:
Lol fair, rephrased.